docs/articles/documentation/extending-testcafe/custom-reporter/reporter-project-commands.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: docs
+title: Reporter Project Commands
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/reporter-project-commands/
+---
+# Reporter Project Commands
```

Comment:
I would rather include the whole page to the `Creating Reporter Plugin` section. There it will fit good into the narrative canvas after the `Implementing Reporter` section without any subtitle:
```
Generating Reporter Project
Implementing Reporter
Building Reporter
Testing Reporter
Previewing Report
Publishing Reporter to NPM
```

BTW, In `Publishing Reporter to NPM` it will be good to include reference to the `Using Reporter` section. And move `Development Version` section from  `Using Reporter` section to the `Implementing Reporter` section and insert it into narrative. Because, if I'm consumer of the already built reporters I don't care about dev version of reporter and this section may confuse me. But if I want to implement reporter I would like to have all required info in one section in form of tutorial.